modal/functions.py

```diff
@@ -8,16 +8,7 @@
 from dataclasses import dataclass
 from datetime import date, timedelta
 from pathlib import Path
-from typing import (
-    Any,
-    AsyncIterable,
-    Callable,
-    Collection,
-    Dict,
-    List,
-    Optional,
-    Union,
-)
+from typing import Any, AsyncIterable, Callable, Collection, Dict, List, Optional, Union
```

Comment:
This line is 88 chars now, and our project configured 120. I dunno why it was line wrapped before.